.github/ISSUE_TEMPLATE/documentation-issue-suggestion.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation issue/suggestion
+about: Report issues or make suggestions about documentation
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**What are issue or suggestion are you reporting?**
```

Comment:
```suggestion
**What issue or suggestion are you reporting?**
```

Although I would just say "Description" in this title.

Reply:
Thanks! I think I softened the language a bit.  Let me know what you think.